src/main/docker/Dockerfile

```diff
@@ -59,6 +56,12 @@ RUN chgrp 0 /usr/local/bin/docker-entrypoint.sh && \
 COPY scripts/ /opt/elasticsearch/scripts/
 RUN chmod +x /opt/elasticsearch/scripts/*
 
+# Configure logging
+COPY /maven/caf-*.jar /usr/share/elasticsearch/lib/
+COPY /maven/commons-*.jar /usr/share/elasticsearch/lib/
+COPY /maven/util-*.jar /usr/share/elasticsearch/lib/
```

Comment:
What are we trying to exclude here?  Why isn't it just all dependencies?  If an exclusion is required would it not be better to do it in the pom file using the normal excludes construct?

Reply:
Excluded the jackson-core artifact in dependencies